### PR TITLE
jdupes: add page

### PR DESCRIPTION
**Pull request by einverne**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/jdupes.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Search all directories recursively:
+
+`jdupes --recurse {{directory}}`
+
+- Search directory recursively and let user choose files to preserve:
+
+`jdupes --delete --recurse {{directory}}`
+
+- Search multiple directories, and follow subdirectores under directory2, not directory1:
```
Comment (zdroid) [inline pages/common/jdupes.md]:
```suggestion
- Search multiple directories and follow subdirectores under directory2, not directory1:
```

File: pages/common/jdupes.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# jdupes
+
+> A powerful duplicate file finder and an enhanced fork of 'fdupes'.
```
Comment (zdroid) [inline pages/common/jdupes.md]:
```suggestion
> A powerful duplicate file finder and an enhanced fork of fdupes.
```

File: pages/common/jdupes.md [hunk 3]
```diff
@@ -19,7 +19,7 @@
 
 `jdupes --delete --recurse {{directory}}`
 
-- Search multiple directories, and follow subdirectores under directory2, not directory1:
+- Search multiple directories and follow subdirectores under directory2, not directory1:
```
Comment (zdroid) [inline pages/common/jdupes.md]: For some reason, directory2 and directory1 appear odd here. Is there a better alternative to be put in their place?

Comment (sbrl) [reply pages/common/jdupes.md]: Not sure I'm seeing the oddness.

**Pull request merged**